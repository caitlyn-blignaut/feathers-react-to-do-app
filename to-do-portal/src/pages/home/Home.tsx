import { Col, Container, Row } from "react-bootstrap";
import { IconContext } from "react-icons";
import { PiListChecksLight } from "react-icons/pi";

export const Home = () => {
  return (
    <Container className="d-flex align-items-center justify-content-center">
      <Row>
        <Col style={{ marginTop: "50px", alignContent: "center" }}>
          <h1 style={{ marginLeft: "20px" }}>To Do</h1>
          <IconContext.Provider value={{ size: "150px" }}>
            <PiListChecksLight />
          </IconContext.Provider>
        </Col>
      </Row>
    </Container>
  );
};
