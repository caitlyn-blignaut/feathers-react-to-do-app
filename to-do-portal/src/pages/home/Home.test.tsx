import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { Home } from "./Home";

describe("Home", () => {
  it("renders the Home component", () => {
    const home = render(<Home />);

    expect(home).toMatchSnapshot();
  });
});
