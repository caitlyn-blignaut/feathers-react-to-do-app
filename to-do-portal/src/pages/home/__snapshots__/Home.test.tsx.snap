// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > renders the Home component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="d-flex align-items-center justify-content-center container"
      >
        <div
          class="row"
        >
          <div
            class="col"
            style="margin-top: 50px; align-content: center;"
          >
            <h1
              style="margin-left: 20px;"
            >
              To Do
            </h1>
            <svg
              fill="currentColor"
              height="150px"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 256 256"
              width="150px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M222,128a6,6,0,0,1-6,6H128a6,6,0,0,1,0-12h88A6,6,0,0,1,222,128ZM128,70h88a6,6,0,0,0,0-12H128a6,6,0,0,0,0,12Zm88,116H128a6,6,0,0,0,0,12h88a6,6,0,0,0,0-12ZM83.76,43.76,56,71.51,44.24,59.76a6,6,0,0,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Zm0,64L56,135.51,44.24,123.76a6,6,0,1,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Zm0,64L56,199.51,44.24,187.76a6,6,0,0,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="d-flex align-items-center justify-content-center container"
    >
      <div
        class="row"
      >
        <div
          class="col"
          style="margin-top: 50px; align-content: center;"
        >
          <h1
            style="margin-left: 20px;"
          >
            To Do
          </h1>
          <svg
            fill="currentColor"
            height="150px"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 256"
            width="150px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M222,128a6,6,0,0,1-6,6H128a6,6,0,0,1,0-12h88A6,6,0,0,1,222,128ZM128,70h88a6,6,0,0,0,0-12H128a6,6,0,0,0,0,12Zm88,116H128a6,6,0,0,0,0,12h88a6,6,0,0,0,0-12ZM83.76,43.76,56,71.51,44.24,59.76a6,6,0,0,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Zm0,64L56,135.51,44.24,123.76a6,6,0,1,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Zm0,64L56,199.51,44.24,187.76a6,6,0,0,0-8.48,8.48l16,16a6,6,0,0,0,8.48,0l32-32a6,6,0,0,0-8.48-8.48Z"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
